infra/object-cache/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM minio/minio:RELEASE.2020-03-25T07-03-04Z
+CMD ["minio", "gateway", "s3", "https://sfo2.digitaloceanspaces.com"]
```

Comment:
Why not put this command into the Compose file?

Reply:
We're deploying via kubernetes, not compose. Let's use docker's automation to handle our daemon entry points.